src/smc_sagews/smc_sagews/tests/test_doctests.py

```diff
@@ -0,0 +1,86 @@
+# run doctests for selected sagemath source files in sagews server
+import socket
+import conftest
+import os
+import re
+
+from textwrap import dedent
+
+
+import pytest
+import future
+import doctest
+
+# doctests need sagemath env settings
+# skip this entire module if not set
+dtvar = 'SAGE_LOCAL'
+if not dtvar in os.environ:
+    pytest.skip("skipping doctests, {} not defined".format(dtvar), allow_module_level=True)
+
+
+@pytest.mark.parametrize("src_file", [
+    ('/ext/sage/sage/local/lib/python2.7/site-packages/sage/symbolic/units.py'),
```

Comment:
aha, consequently, maybe `SAGE_LOCAL = os.environ('SAGE_LOCAL')` and then `join(SAGE_LOCAL, 'local/lib/python2.7/site-packages/sage/symbolic/units.py')` ? just a thought …

Reply:
my general idea is to customize sage local, because we have several ones

Reply:
Is that needed now, or can it be added as a next step?

I chose SAGE_LOCAL at random from the many vars set in `sage-env`, simply as a heuristic test whether the script had been run at all. There are dependencies on a few SAGE_* variables in the doctest. It fails in surprising ways if you try to run test_doctest.py without them.

Reply:
ah, I have no idea, actually. well, I've changed that script to respect any SAGE_LOCAL being set. I don't know much about these env vars. My only thoughts are to be able to run tests for more than one sage setup (and not mix parts of one with another one). Let's merge this, it's 0-risk anyways.